src/api/alchemy-contract.ts

```diff
@@ -0,0 +1,3 @@
+import { Contract as EthersContract } from '@ethersproject/contracts';
+
+export class Contract extends EthersContract {}
```

Comment:
The subclass here is because we plan to later add or tweak stuff in it?

Reply:
we're going to eventually add overloads to pass in the `Alchemy` instance into the constructor instead of the needing to pass in the provider